src/Keboola/DbExtractor/Application.php

```diff
@@ -122,35 +123,6 @@ private function validateTableParameters(array $table): void
         }
     }
 
-    private function validateParameters(array $parameters): array
```

Comment:
Tohle prosím přesuň na původní místo, t je vidět diff (jestli tam nějaký je). Reviewnu to až budou procházet testy.

Reply:
Přesunuto